src/main/java/mate/academy/security/AuthenticationServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.security;
+
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
+import mate.academy.model.User;
+
+public class AuthenticationServiceImpl implements AuthenticationService {
+    @Override
+    public User login(String email, String password) throws AuthenticationException {
+        return null;
```

Comment:
The `login` method is currently returning `null`. Implement the logic to authenticate a user by verifying the provided email and password. If authentication fails, throw an `AuthenticationException`.